playbooks/roles/wordpress/templates/default-nginx.j2

```diff
@@ -0,0 +1,31 @@
+server {
+	listen *:80;
+	server_name {{ DEFAULT_HOSTNAME }};
+
+	root /var/www/{{ DEFAULT_HOSTNAME }};
+	index index.html index.php;
+
+        access_log  /var/www/log/{{ DEFAULT_HOSTNAME }}_nginx_access.log;
+        error_log /var/www/log/{{ DEFAULT_HOSTNAME }}_nginx_error.log error;
+
+	location / {
+		try_files $uri $uri/ =404;
+	}
+
+	location ~ [^/]\.php(/|$) {
+		fastcgi_split_path_info ^(.+?\.php)(/.*)$;
+		if (!-f $document_root$fastcgi_script_name) {
+			return 404;
+		}
+
+		fastcgi_pass unix:/var/run/php5-fpm.sock;
```

Comment:
@Agrendalath, it might be a good idea to use create and separate FPM pools for each site.

Reply:
Hm, I don't think we actually need it here. The `default` page doesn't have any PHP files. I'll just remove it.